js/screens/Login/styles.js

```diff
@@ -24,52 +25,52 @@ const styles = StyleSheet.create({
     margin: 0,
     fontSize: 70,
     fontWeight: '600',
-    fontFamily: 'BalooBhai-Regular',
+    fontFamily: typography.mainFont,
   },
   subHeader: {
     transform: [{ translateY: -45 }],
```

Comment:
We should not do translate in negative. Please try to come up with another solution.